### DISCUSSION TITLE
Hovering right editor sidebar make a "pop-up" bug

### DISCUSSION THREAD

**Discussion by alcgabriel** [ppy/osu#30493] [Q&A] [answered]
Like the box tries to fit somewhere

https://github.com/user-attachments/assets/c782e6a9-33eb-4c5c-86c5-4eab70c31699

using Wayland/Hyprland(TWM)

logs
[compressed-logs.zip](https://github.com/user-attachments/files/17613185/compressed-logs.zip)

**Comment by Joehuu** [comment, marked as answer]
I've already fixed this in https://github.com/ppy/osu/pull/30450, just need to wait next release.
